src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,107 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPASITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPASITY];
+    }
+
+    private class Node<K, V>{
```

Comment:
You can move inner class at the end of the class

Reply:
I think this is optional. It can be either at the beginning or at the end. Earlier I wrote in the same way and the mentors did not say anything.

Reply:
No problem